### PR TITLE
Added Motion Blur

### PR DESCRIPTION
**Pull request by pavelstencl**
Hi, thank u for your work. It is awesome! Could u pls add this feature to your branch? Thanks :)

### CODE REVIEW

File: src/magick-image.ts [hunk 1]
```diff
@@ -205,6 +205,8 @@ export interface IMagickImage extends IDisposable {
     modulate(brightness: Percentage): void;
     modulate(brightness: Percentage, saturation: Percentage): void;
     modulate(brightness: Percentage, saturation: Percentage, hue: Percentage): void;
+    motionBlur():void;
+    motionBlur(radius?: number, sigma?: number, angle?: number): void;
```
Comment (dlemstra) [inline src/magick-image.ts:209]: All these numbers should be mandatory in this overload.

File: src/magick-image.ts [hunk 2]
```diff
@@ -205,6 +205,8 @@ export interface IMagickImage extends IDisposable {
     modulate(brightness: Percentage): void;
     modulate(brightness: Percentage, saturation: Percentage): void;
     modulate(brightness: Percentage, saturation: Percentage, hue: Percentage): void;
+    motionBlur():void;
```
Comment (dlemstra) [inline src/magick-image.ts:208]: Can you please remove this overload? The user will always need to specify the values. In ImageMagick the last value is not mandatory but I don't want to create a specific overload for this. And this overload is also not available in the Magick.NET api and I am trying to keep this library as close as possible to the Magick.NET library.

File: tests/magick-image/motion-blur.spec.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+// Copyright Dirk Lemstra https://github.com/dlemstra/magick-wasm.
+// Licensed under the Apache License, Version 2.0.
+
+import { IMagickImage, MagickImage } from '../../src/magick-image';
+
+let image: IMagickImage;
```
Comment (dlemstra) [inline tests/magick-image/motion-blur.spec.ts:6]: You don't need this anymore. You can use `TestImages.Builtin.logo.use((image) => {` in the unit test instead.

**Review by dlemstra — COMMENTED** [review]
Thanks for helping about. Left some comments on things I would like to see changed.

**Comment by pavelstencl** [comment]
Thanks for CR, i pushed updated version with all your notes implemented.

**Review by dlemstra — APPROVED** [review]
Thanks for helping me out.

**Pull request merged**